insecure/attacker.pb.go

```diff
@@ -136,82 +136,37 @@ func (m *Message) GetProtocol() Protocol {
 	return Protocol_UNKNOWN
```

Comment:
💡 this file is auto-generated by `protobuf`, hence may be skipped while reviewing.